packages/react-core/src/index.ts

```diff
@@ -1,2 +1,258 @@
 // main entry point = evm
-export * from "./evm";
+
```

Comment:
this seems like a lot of duplication, we're already doing this in `/evm/index.ts` no? or does doing export * from `./evm` break tree shaking or something?

Reply:
Agree, It's a huge pain. I wanted to see whether it helps with tree-shaking or not but did not test it.

I tested the `export *` vs named exports now - looks like both have the tree-shaking issue. As soon as something from `@thirdweb-dev/sdk` or `@thirdweb-dev/auth` is included in export - it instantly kills the tree-shaking. 

Example: 

<img width="1012" alt="image" src="https://user-images.githubusercontent.com/22043396/224274196-108fb2b0-c4aa-4d7c-a1ae-cd621ef07ecb.png">